src/marks/waffle.js

```diff
@@ -104,6 +104,7 @@ function waffleRender(y) {
           )
           .attr("fill", (i) => `url(#${patternId}-${i})`)
           .attr("stroke", this.stroke == null ? null : (i) => `url(#${patternId}-${i})`)
+          .call(applyChannelStyles, this, {href})
```

Comment:
I think ariaLabel and title should travel with href, too. We want the visual channels to apply to the pattern, but the non-visual (semantic?) channels should apply to the path.